include/deal.II/fe/mapping.h

```diff
@@ -488,23 +485,24 @@ class Mapping : public Subscriptor
 
 protected:
   /**
-   * Indicate fields to be updated in the constructor of FEValues. Especially,
-   * fields not asked for by FEValues, but computed for efficiency reasons
-   * will be notified here.
+   * Given a set of update flags, compute which other quantities <i>also</i>
+   * need to be computed in order to satisfy the request by the given flags.
+   * Then return the combination of the original set of flags and those
+   * just computed.
    *
-   * See
-   * @ref UpdateFlagsEssay.
-   */
-  virtual UpdateFlags update_once (const UpdateFlags) const = 0;
-
-  /**
-   * The same as update_once(), but for the flags to be updated for each grid
-   * cell.
+   * As an example, if @p update_flags contains update_JxW_values (i.e., the
+   * product of the determinant of the Jacobian and the weights provided by
+   * the quadrature formula), most mappings will require the computation of
+   * the full Jacobian matrix in order to compute its determinant. They
+   * would then return not just update_JxW_values, but also
+   * update_jacobians.
```

Comment:
I think this example is incorrect. If I remember things correctly, we use update_contravariant_transformation internally. Of course, this is the Jacobian of the transformation, but if we set the flag update_jacobians we additionally copy the data into separate fields. One might argue if that is really necessary, but that's how things work. A better example would be second derivatives which also need first derivatives thanks to Hamed Maien.